### PR TITLE
server: update lb rule with new protocol

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes the issue that update lb rule with new procol does not work, as it is not implemented yet in server side.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [x] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: server/src/main/java/com/cloud/network/lb/LoadBalancingRulesManagerImpl.java [hunk 1]
```diff
@@ -2133,6 +2134,10 @@ public LoadBalancer updateLoadBalancerRule(UpdateLoadBalancerRuleCmd cmd) {
             lb.setDisplay(forDisplay);
         }
 
+        if (lbProtocol != null) {
```
Comment (nvazquez) [inline server/src/main/java/com/cloud/network/lb/LoadBalancingRulesManagerImpl.java:2137]: Maybe use StringUtils.isNotBlank() instead?

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan package

**Review by GutoVeronezi — APPROVED** [review]

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Review by sureshanaparti — APPROVED** [review]
code LGTM

**Comment by sureshanaparti** [comment]
@blueorangutan test

**Pull request merged**